src/style.css

```diff
@@ -2,2 +1,66 @@
+html {
+  --p-header: 50px;
+  --active-color: #00acdc;
+}
+
+body,
+li,
+ul,
+a {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
   margin: 0;
+  padding: 0;
+  color: #000;
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 var(--p-header);
+  width: calc(100% - (var(--p-header) * 2));
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__menu {
+  display: flex;
+  list-style: none;
+}
+
+.nav__menu__item:not(:last-child) {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__menu__item a {
+  display: flex;
+  align-items: center;
+  height: 60px;
```

Comment:
The height of the anchor tags is set to 60px. Ensure that this fixed height does not cause layout issues on smaller screens or with larger text sizes.